starwars_api/models.py

```diff
@@ -11,15 +11,21 @@ def __init__(self, json_data):
         Dynamically assign all attributes in `json_data` as instance
         attributes of the Model.
         """
-        pass
+        for key, value in json_data.items():
+            setattr(self, key, value)
+            # setattr(x, 'foobar', 123) is equivalent to x.foobar = 123
```

Comment:
@jangeador yup. we used it for `def get(cls, resource_id)` (scroll down). your thoughts? anywhere else you'd recommend using it for this project?